feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -55,42 +66,65 @@ fun ProfileView(
     )
 }
 
+/**
+ * Function [ProfileContainer] contains the actual view of ProfileView.
+ * It will generate the composable elements to create the user's profile.
+ * @param profileValues - to have the profile values passed from View Model
+ * @param onClickEditProfileButton - high order function to handle action when clicking on Edit button
+ * @param onClickPaymentMethodsButton - high order function to handle action when clicking on Payment Method button
+ * @param onClickEmergencyContactsButton - high order function to handle action when clicking on Emergency button
+ * @param onClickLogOutButton - high order function to handle action when clicking on Log Out button
+ *
+ * @author Juan Islas
+ * @modifiedBy Jose Miguel Garcia Reyes
+ */
+
 @Composable
 fun ProfileContainer(
     profileValues: Profile,
```

Comment:
Rename as **profileValues** to **profileValues**

Reply:
Do you mean only "profile"? Done :)